### ISSUE TITLE
Add Swipeable Card

### ISSUE THREAD

**Issue by michelelacorte** [lightSky/Awesome-MaterialDesign#22]
Could you add Swipeable Card library?

https://github.com/michelelacorte/SwipeableCard

Thanks,
Best regards.

**Comment by lightSky** [comment]
Yes,of course,the swipeableCard is nice .I think it can be classified  to "view" part,how do you think ?And you can upload representative gifs and create pull request .

Thanks for you contribution.

**Comment by michelelacorte** [comment]
Thanks @lightSky I've create a pull request and I add SwipeableCard under View section
